apps/coscrad-frontend/src/utils/generic-components/presenters/flat-multilingual-text-presenter.tsx

```diff
@@ -0,0 +1,49 @@
+import { IMultilingualTextItem } from '@coscrad/api-interfaces';
+import { Box, Paper, styled } from '@mui/material';
+import { MultilingualTextItemPresenter } from './multilingual-text-item-presenter';
+
+const MultilingualTextBox = styled(Paper)({
+    paddingLeft: '8px',
+});
+
+interface FlatMultilingualTextPresenterProps {
+    primaryMultilingualTextItem: IMultilingualTextItem;
+    translations: IMultilingualTextItem[];
+}
+
+export const FlatMultilingualTextPresenter = ({
+    primaryMultilingualTextItem,
+    translations,
+}: FlatMultilingualTextPresenterProps): JSX.Element => {
+    const { text, languageCode, role } = primaryMultilingualTextItem;
+
+    return (
+        <MultilingualTextBox
```

Comment:
I'll have to take a look at this via Cypress or local run to see how it looks. This is great!